src/types.ts

```diff
@@ -9,6 +9,24 @@ export type StructureSchema = {
   [key: string]: StructureProperty;
 };
 
+export interface ToolParameter {
+  type: "string" | "integer" | "number" | "boolean" | "object" | "array";
```

Comment:
```suggestion
  type: "string" | "integer" | "number" | "boolean" | "object";
```
I didn't handle arrays yet